stripe/src/main/java/com/stripe/android/view/PaymentAuthWebView.kt

```diff
@@ -107,9 +110,10 @@ internal class PaymentAuthWebView @JvmOverloads constructor(
     }
 
     internal class PaymentAuthWebViewClient(
-        private val activity: Activity,
```

Comment:
Could you elaborate the reason for this change? If we are trying to breaking activity dependency from `PaymentAuthWebViewClient`, should we do it from `PaymentAuthWebView#init` ?

Reply:
I want to create better tests that use real Android objects. This also sets us up for further refactoring.